python2/primes.py

```diff
@@ -0,0 +1,13 @@
+def is_prime(num):
+    """Returns True if the number is prime
+    else False."""
+    if num == 0 or num == 1:
+        return False
+    for x in range(2, num):
```

Comment:
Good point, thanks
On Jun 18, 2016 15:22, "Deniz Kennedy" <notifications@github.com> wrote:

> In python2/primes.py
> <https://github.com/naslundx/language-benchmarker/pull/23#discussion_r67600920>
> :
>
> > @@ -0,0 +1,13 @@
> > +def is_prime(num):
> > +    """Returns True if the number is prime
> > +    else False."""
> > +    if num == 0 or num == 1:
> > +        return False
> > +    for x in range(2, num):
>
> Is it range or xrange?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/naslundx/language-benchmarker/pull/23/files/2c8d356e4028779c9f72cc14c3862ede2f3475a0#r67600920>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACFvhILy9nwEV_yI7bE8EY1_finnHiIFks5qM_EmgaJpZM4I47ES>
> .
>